modules/grafana-agent-tempo/variables.tf

```diff
@@ -28,3 +28,8 @@ variable "tempo_attributes" {
   default     = {}
   description = "Attributes to set for all endpoints."
 }
+variable "k8s_namespace" {
+  type        = string
+  default     = "default"
+  description = "Kubernetes namespace to use"
```

Comment:
This is only for the DaemonSet / Deployments - the service accounts cluster role{s, bindings} are not namespaced.